partials/tabs/coalaonline.html

```diff
@@ -146,7 +146,8 @@ <h4>Optional Settings</h4>
         <div class="row blue-grey darken-4 white-text z-depth-1 no-margin">
           <span>
             <a class="waves-effect grey-text text-lighten-2">
-              <i ng-click="add_bears(section)" class="fa fa-plus-circle bears-icon-small" aria-hidden="true"> &nbsp;ADD BEAR
+              <i ng-click="add_bears(section)" class="fa fa-plus-circle bears-icon-small" aria-hidden="true">
+                &nbsp;<span style="font-family:'Roboto';">ADD BEAR</span>
```

Comment:
Great.  You've found the button and changed it.

However, we avoid inline styles.

where is the times new roman coming from?

note that `bears-icon-small` and similar are defined in http://github.com/coala/coalaCSS

Probably the fix needs to occur there.

And probably the solution is not adding `Roboto` to `bears-icon-small`, but requires finding why Times New Roman is being inherited and globally disabling it at the root of the problem.

Reply:
The root of the problem here is the .fa class of the font awesome icon. Since the "ADD BEAR" text has been written within the <i></i> tag, the style "font-family: FontAwesome" automatically gets applied to the text.
At present only the:
h1,
h2,
h3,
h4,
h5,
h6 {
  font-family: Roboto;
}

style is the best option from the http://github.com/coala/coalaCSS file to apply the Roboto font-family to the text. But this causes the styles of the materialize.css file heading tags to be applied automatically too.
![screenshot from 2018-08-15 22-53-24](https://user-images.githubusercontent.com/42298056/44162875-41fc3900-a0df-11e8-8419-8b6eb5004be2.png)

We can define in a new class in the CoalaCSS file to attend to this particular issue. What do you suggest?

Reply:
Yeah, new class in coalaCSS sounds good!